src/grid.scss

```diff
@@ -0,0 +1,119 @@
+@import "./mixins/breakpoints.scss";
+@import "./variables/grid.scss";
+
+/*
+ * Grid classes
+ * Component code   : gd
+ *
+ * Modifier 1       : gap   ( grid gap modifier )
+ *                    col   ( grid column modifier )
+ *                    cols  ( grid columns template modifier )
+ *
+ * Modifier 2       : eq ( grid with equal sized cols ) [Modifier 1: cols]
+
+ *                      Variables used:
+ *                          --gd-cols-eq     ( num cols )
+ *                          --gd-cols-eq-m   ( num cols @m )
+ *                          --gd-cols-eq-l   ( num cols @l )
+ *                          --gd-cols-eq-xl  ( num cols @xl )
+ *                          --gd-cols-eq-2xl ( num cols @2xl )
+ *
+ *                    cus ( grid with custom cols size ) [Modifier 1: cols,col,gap]
+ *
+ *                      Variables used:
+ *                        [Modifier 1: cols]
+ *                          --gd-cols-cus     (num of cus cols )
+ *                          --gd-cols-cus-m   (num of cus cols @m )
+ *                          --gd-cols-cus-l   (num of cus cols @l )
+ *                          --gd-cols-cus-xl  (num of cus cols @xl )
+ *                          --gd-cols-cus-2xl (num of cus cols @2xl )
+ *
+ *                        [Modifier 1: gap]
+ *                          --gd-gap-cus     ( gap size s )
+ *                          --gd-gap-cus-m   ( gap size m )
+ *                          --gd-gap-cus-l   ( gap size l )
+ *                          --gd-gap-cus-xl  ( gap size xl )
+ *                          --gd-gap-cus-2xl ( gap size 2xl )
+ *
+ *                    subgrid (subgrid within col)
+ *
+ *                      Variables used:
+ *                        [Modifier 1: col]
+ *                          --gd-col-cus ( for cus subgrid within a col )
+ *
+ */
+
+.gd-gap-cus-m { gap: $grid-m; }
```

Comment:
SCSS variable can be named in this format. $gd-gap-m would have been good in this case. Consider renaming all the other variables in this same way.